### PR TITLE
Gold Fish Currency Payouts 

### LINKED ISSUE #50: Gold Fish?
Very fun little command-line version 😸 

Looking at your code, I don't see a way for the user to obtain gold fish

No Currency:
https://github.com/SwartzCr/nekoatsume/blob/master/lib/update.py#L112
https://github.com/SwartzCr/nekoatsume/blob/master/lib/display.py#L151

### PR DESCRIPTION
**Pull request by carystanley**
fixes #50

So not sure about the larger cat economy, but this will at least add Gold fish into the mix of cat payouts allowing the user to buy gold fish items they have always wanted. Like the "robotic laser pointer" 😉 

I gave a 1/30 chance that the cat will pay_up in gold fish

Note: This is probably my first time writing python code, so may be incredibly ugly/buggy

### CODE REVIEW

**Comment by SwartzCr** [comment]
Hey! Thanks for the PR!
Have you tested this? It seems to have a ; in it - which last I checked wasn't a thing that python does.
Otherwise it looks mostly good :D

**Comment by carystanley** [comment]
I did test it and it did work, but I have removed the semi-colon as it is rightly not correct python.

File: lib/display.py [hunk 1]
```diff
@@ -144,11 +144,12 @@ def collect_money(data):
...
     for i in range(len(data["pending_money"])):
         money = data["pending_money"].pop()
+        currency = money[2]
         printer.p("{.GREEN}[$$$$$$]{.ENDC}".format(
             printer.PColors,
-            printer.PColors), "Yes! {0} left you {1} fish!".format(
-            money[0], str(money[1])))
-        data["s_fish"] += money[1]
+            printer.PColors), "Yes! {0} left you {1}{2} fish!".format(
+            money[0], str(money[1]), " gold" if currency == "g" else ""))
```
Comment (SwartzCr) [inline lib/display.py:151]: Can you make this `else " silver"`?
Otherwise this is great!!

**Review by SwartzCr — DISMISSED** [review]

**Comment by SwartzCr** [comment]
Hey @carystanley ! If you wanna make one small change I'll merge this!

**Comment by SwartzCr** [comment]
@carystanley I'm going to add the silver text myself - thanks so much for making this PR!

**Pull request merged**